DataFormats/GEMRecHit/interface/ME0Segment.h

```diff
@@ -72,7 +73,10 @@ class ME0Segment final : public RecSegment {
 
   float time() const    { return theTimeValue; }
   float timeErr() const { return theTimeUncrt; }
-    
+
+  float deltaPhi() const { return theDeltaPhi; }
+  static float computeDeltaPhi(const ME0Chamber * chamber, const LocalPoint& position, const LocalVector& direction );
```

Comment:
@ianna If there is no problem with ME0SegmentAlgorithm depending on it, then I think it is fine.

@calabria We also need to calculate it for projected pixel tracks as we compare projected and segment bending angles when matching.